### DISCUSSION TITLE
共享规则不生效，为什么

### DISCUSSION THREAD

**Discussion by XiaotingCui** [steedos/steedos-platform#6080] [Q&A]
根据官方提供的训练营视频练习的，其中共享规则不生效，也不报错，我的配置和平台版本如下，麻烦各位大佬帮忙解答
![1703828889502](https://github.com/steedos/steedos-platform/assets/70877207/2dfd1624-7970-468a-ade7-fe80e61d42fc)
![1703828936039](https://github.com/steedos/steedos-platform/assets/70877207/d37ce301-f191-4f0a-a1ae-57310386f2ec)

**Comment by hotlong** [comment]
共享规则的公式不能用amis语法，参考这个文档 https://docs.steedos.cn/zh-CN/no-code/customize/formula/overview
